antares.external.diagnostics.compilerhost.nuspec

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>antares.external.diagnostics.compilerhost</id>
+    <version>1.0.19</version>
+    <authors>Shekhar Gupta</authors>
+    <owners>shgup</owners>
```

Comment:
We dont need these files anymore in master2 as we are not going to generate antares external nuget packages in Liberation project.